CHANGES.rst

```diff
@@ -22,7 +22,7 @@ astropy-helpers Changelog
 2.0.4 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Support dotted package names as namespace packages. [#370]
```

Comment:
Maybe say ``in generate_version_py``

Reply:
OK, will do it before merge, but now rather wait for the CI to finish.